### ISSUE TITLE
Question on rum internals in relation to rules.

### ISSUE THREAD

**Issue by drewverlee** [oakes/odoyle-rum#1]
In the case of ssr, is just the components that were scheduled that get re rendered? Or is it the entire page? I used to think it was the later, but given your using it here I have to assume it's the former.

**Comment by oakes** [comment]
I don't understand the question. For server side rendering there isn't really a concept of re-rendering. It just runs a single time to generate the markup. Re-rendering can only occur on the client side when rules fire after the initial mount. If that doesn't answer the Q can you rephrase it?

**Comment by drewverlee** [comment]
Thanks. 

I was able to clear up my confusion, as you say here, the HTML & CSS is only generated once on the server-side on initial load. I was under the impression that happened every update.
